disk_element.py

```diff
@@ -0,0 +1,336 @@
+import warnings
+import numpy as np
+import pandas as pd
+import scipy.interpolate as interpolate
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+from ross.data_io.read_xl import (
+    load_bearing_seals_from_yaml,
+    load_bearing_seals_from_xltrc,
+    load_disks_from_xltrc,
+    load_shaft_from_xltrc,
+)
+
+__all__ = [
+    "LumpedDiskElement",
+    "DiskElement",
+]
+
+class Element:
+    """Element class."""
+
+    def __init__(self):
+        pass
+
+    def summary(self):
+        """A summary for the element.
+        A pandas series with the element properties as variables.
+        """
+        attributes = self.__dict__
+        attributes["type"] = self.__class__.__name__
+        return pd.Series(attributes)
+
+class LumpedDiskElement(Element):
```

Comment:
We will have only one class for the disks with one additional constructor for geometry arguments:

```python
class DiskElement(Element):
    def __init__(self, m, Id, Ip):
        ...
    
    @classmethod
    def from_geometry(cls, n, material, width, i_d, o_d):
        ...
        return cls(n, m, Id, Ip)
```